darwin/dataset/remote_dataset.py

```diff
@@ -251,16 +255,46 @@ def pull(
             If the given ``release`` has an invalid format.
         ValueError
             If darwin in unable to get ``Team`` configuration.
+        ValueError
+            If the release is still processing after the maximum retry duration.
         """
 
         console = self.console or Console()
 
         if release is None:
-            release = self.get_release()
+            if retry:
+                raise ValueError(
+                    "To retry downloading a release, a release name must be provided. This can be done as follows:\n\nrelease = dataset.get_release(name='release_name')\ndataset.pull(release=release, retry=True)"
```

Comment:
[As above](https://github.com/v7labs/darwin-py/pull/876/files#r1658843008), with an alternative error message tailored to the SDK method of pulling